### DISCUSSION TITLE
Issue template for user support?

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1622] [Team Posts]
Hi,
Should we add the template for the users requesting help?

![image](https://user-images.githubusercontent.com/5743546/116328828-d3b32280-a804-11eb-847d-8dd7e17bcffa.png)

Currently, content of "Question" template is empty, and I realized that _almost all of_ users asking help does not know what they should include in the initial post, such as: The proceduce to reproduce the problem; system log during the problematic behavior; configuration they are using.

**Comment by racke** [comment]
IMHO we can't really determine which information is needed for the "question". Logs and configuration might be irrevelant to the question. The only thing that is often relevant is the Sympa version. 

See https://github.com/sympa-community/sympa/issues/1162, the image and the issue title were sufficient to answer the question.

**Comment by ikedas** [comment]
"Question" template may be left empty to be used for general questions.

I think we could add a new template for user support.   That way, we can put in fields specific to user support,  even if the user doesn't fill in all of the fields.  I think it'd better to include at least the same fields for "Bug" template: Version, Installation method, Expected and actual behavior, Additional information.

**Comment by ikedas** [comment]
I added the same content as "Bug" template to the "Question" template.
